doc/user-guides/multi-cluster.md

```diff
@@ -0,0 +1,544 @@
+
+- Completed the Kuadrant Install Guide for at least two clusters [Install Guide](../install/install-openshift.md)
+- kubectl command line tool
+- (optional) have user workload monitoring configured to remote write to a central storage system such as Thanos (also covered in the installation guide).
+
+## Overview
+
+In this doc we will walk you through using Kuadrant to secure, protect and connect an API via a set of Gateways distributed across multiple clusters. 
+We will take the approach of assuming certain personas and how they can each work with Kuadrant to achieve their goals.
+
+### What Kuadrant can do for you in a multi-cluster environment
+
+Kuadrant's capabilities can be leveraged in single or multiple clusters. Below is a list of features that are designed to work across multiple clusters as well as in a single cluster environment.
+
+- **Multi-Cluster Ingress:** Kuadrant provides multi-cluster ingress connectivity using DNS to bring traffic to your Gateways using a strategy defined in a `DNSPolicy` (more later). 
+- **Global Rate Limiting:** Kuadrant can enable global rate limiting usecases when it is configured to use a shared store (redis) for counters based on limits defined by a `RateLimitPolicy`.
+- **Global Auth:*** Kuadrant's `AuthPolicy` can be configured to leverage external auth providers to ensure different cluster exposing the same API are authenticating and authorizing in the same way. 
+- **Integration with federated metrics stores:** Kuadrant has example dashboards and metrics that can be used for visualizing your gateways and observing traffic hitting those gateways across multiple clusters. 
+
+**Platform Engineer**
+
+We will walk through deploying a gateway that provides secure communications and is protected and ready to be used by development teams to deploy an API. We will then walk through how you can have this gateway in clusters in different geographic regions and leverage Kuadrant to bring the specific traffic to your geo located gateways to reduce latency and distribute load while still having it protected and secured via global rate limiting and auth.
+
+As an optional extra we will highlight how, with the user workload monitoring observability stack deployed, these gateways can then be observed and monitored. 
+
+**Developer**
+
+We will walk through how you can use the kuadrant OAS extensions and CLI to generate a `HTTPRoute` for your API and add both Auth and Rate Limiting to your API.
+
+## Platform Engineer
+
+The following steps should be done in each cluster individually unless specifically excluded. 
+
+### Environment Variables
+
+For convenience in this guide we use some env vars throughout this document
+
+```bash
+export zid=change-this-to-your-zone-id
+export rootDomain=example.com
+export gatewayNS=api-gateway
+export gatewayName=external
+export AWS_ACCESS_KEY_ID=xxxx
+export AWS_SECRET_ACCESS_KEY=xxxx
+export clusterIssuerName=lets-encrypt
```

Comment:
There are two other `lets-encrypt` occurrences apart from this one in the doc and believe these should be replaced with this env var too.